src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int COUNT_FIGURE = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = "white";
+    private static final double MAX_VALUE = 30;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figure = random.nextInt(COUNT_FIGURE);
+        switch (figure) {
+            case 0:
+                return getRandomCircle();
+            case 1:
+                return getRandomIsoscelesTrapezoid();
+            case 2:
+                return getRandomRectangle();
+            case 3:
+                return getRandomRightTriangle();
+            case 4:
+                return getRandomSquare();
+            default:
+                return getDefaultFigure();
+        }
+    }
+
+    public Circle getRandomCircle() {
+        String color = colorSupplier.getRandomColor();
+        double radius = random.nextDouble() * MAX_VALUE;
```

Comment:
Let’s  create a private method that returns a random value and avoid  code duplicates